File: app/src/main/java/com/cashzhang/nozdormu/RxUtils.java
package com.cashzhang.nozdormu;

import io.reactivex.Observable;
import io.reactivex.Observer;
import io.reactivex.android.schedulers.AndroidSchedulers;
import io.reactivex.schedulers.Schedulers;

public class RxUtils {
    public static void CustomSubscribe(Observable observable, Observer observer) {
        observable.subscribeOn(Schedulers.newThread())
                .observeOn(AndroidSchedulers.mainThread())
                .subscribe(observer);
    }
}

File: app/src/main/java/com/cashzhang/nozdormu/RxUtils.kt
package com.cashzhang.nozdormu

import io.reactivex.Observable
import io.reactivex.Observer
import io.reactivex.android.schedulers.AndroidSchedulers
import io.reactivex.schedulers.Schedulers

object RxUtils {
    @JvmStatic
    fun CustomSubscribe(observable: Observable<*>, observer: Observer<*>?) {
        observable.subscribeOn(Schedulers.newThread())
                .observeOn(AndroidSchedulers.mainThread())
                .subscribe(observer)
    }
}